src/App/Models/NotificationData.cs

```diff
@@ -3,21 +3,20 @@ namespace Bit.App.Models
 {
     public abstract class BaseNotificationData
     {
-        public abstract string Type { get; }
+        public abstract string NotificationType { get; }
 
+        public string NotificationId { get; set; }
```

Comment:
IMHO there is no need to add the prefix "Notification" to the properties given that the class is called notification data so adding the prefix to anything in there is redundant. Or was is confusing on some context?
Also, for now we can leave it as is but in the future if there are notifications that don't require additional data we should extract the `Id` out of this class (which is the only thing required).